Rewrite/src/Rewrite.CSharp/Tree/Cs.cs

```diff
@@ -28,7 +28,7 @@ bool Core.Tree.IsAcceptable<R, P>(ITreeVisitor<R, P> v, P p)
     {
         return v.DefaultValue(this, p);
     }
-
+#pragma warning disable CS0108
```

Comment:
This file is all generated code. We could have this warning disabled for all class headers. Would that be OK?

Reply:
This is due to it it being declared on interface type and CS inheriting from J, as the definition exists on both `J` and `CS`. It is essentially complaining that you defined the same class type on both parent interface and child interface. While this is syntactically allowed, it is discouraged and generally you need to use special syntax when you wanna do this deliberately. Interestingly enough MSBuild flags this as a warning (asking for `new` keyword to be used to ensure this is deliberate) BUT Rider is the opposite, says `new` keyword is not needed. Since I couldn't reconcile this behavior I just chose to disable it. I don't want to disable it at file level as it will eliminate this check for not just types but members. 

I think it requires discussion on best way to handle it.